### PR TITLE
Add Rep. Mary Peltola

### PR DESCRIPTION
**Pull request by putorti**
Sworn in on September 13, 2022

### CODE REVIEW

File: legislators-current.yaml [hunk 1]
```diff
@@ -40919,3 +40919,26 @@
     district: 1
     party: Republican
     phone: 202-225-2472
+- id:
+    bioguide: F000475
```
Comment (JoshData) [inline legislators-current.yaml:40923]: That's not right. :)

Comment (putorti) [reply legislators-current.yaml:40923]: Sorry about that, good now?

Comment (dwillis) [reply legislators-current.yaml:40923]: @putorti The `fecid` is the candidate ID, not the committee ID. Can you fix?

Comment (putorti) [reply legislators-current.yaml:40923]: @dwillis Sorry, my first new official; fewer mistakes next time!

**Comment by JoshData** [comment]
Hijacking your branch a bit :), I made some additional corrections and also added the two other reps sworn in last week. @dwillis Would you look it over?

**Comment by dwillis** [comment]
Looks good to me - good idea to add a validator for FEC IDs, @JoshData .

**Review by dwillis — APPROVED** [review]

**Pull request merged**